src/locales/de_DE.json

```diff
@@ -72,59 +85,60 @@
 "label.action.delete.nic": "NIC entfernen",
-"label.action.delete.physical.network": "Physikalisches Netzwerk l\u00f6schen",
-"label.action.delete.pod": "Pod l\u00f6schen",
-"label.action.delete.pod.processing": "Pod wird gel\u00f6scht....",
-"label.action.delete.primary.storage": "Hauptspeicher l\u00f6schen",
-"label.action.delete.primary.storage.processing": "Hauptspeicher wird gel\u00f6scht....",
-"label.action.delete.secondary.storage": "Sekund\u00e4rspeicher l\u00f6schen",
-"label.action.delete.secondary.storage.processing": "Sekund\u00e4rspeicher wird gel\u00f6scht....",
-"label.action.delete.security.group": "Sicherheitsgruppe l\u00f6schen",
-"label.action.delete.security.group.processing": "Sicherheitsgruppe wird gel\u00f6scht....",
-"label.action.delete.service.offering": "Dienstangebot l\u00f6schen",
-"label.action.delete.service.offering.processing": "Dienstangebot wird gel\u00f6scht....",
-"label.action.delete.snapshot": "Schnappschuss l\u00f6schen",
-"label.action.delete.snapshot.processing": "Schnappschuss wird gel\u00f6scht....",
-"label.action.delete.system.service.offering": "Systemdienstangebot l\u00f6schen",
-"label.action.delete.template": "Vorlage l\u00f6schen",
-"label.action.delete.template.processing": "Vorlage wird gel\u00f6scht....",
-"label.action.delete.user": "Benutzer l\u00f6schen",
-"label.action.delete.user.processing": "Benutzer wird gel\u00f6scht....",
-"label.action.delete.volume": "Volumen l\u00f6schen",
-"label.action.delete.volume.processing": "Volumen wird gel\u00f6scht....",
-"label.action.delete.zone": "Zone l\u00f6schen",
-"label.action.delete.zone.processing": "Zone wird gel\u00f6scht....",
+"label.action.delete.physical.network": "Physikalisches Netzwerk löschen",
+"label.action.delete.pod": "Pod löschen",
+"label.action.delete.pod.processing": "Pod wird gelöscht....",
+"label.action.delete.primary.storage": "Hauptspeicher löschen",
+"label.action.delete.primary.storage.processing": "Hauptspeicher wird gelöscht....",
+"label.action.delete.secondary.storage": "Sekundärspeicher löschen",
+"label.action.delete.secondary.storage.processing": "Sekundärspeicher wird gelöscht....",
+"label.action.delete.security.group": "Sicherheitsgruppe löschen",
+"label.action.delete.security.group.processing": "Sicherheitsgruppe wird gelöscht....",
+"label.action.delete.service.offering": "Dienstangebot löschen",
+"label.action.delete.service.offering.processing": "Dienstangebot wird gelöscht....",
+"label.action.delete.snapshot": "Schnappschuss löschen",
+"label.action.delete.snapshot.processing": "Schnappschuss wird gelöscht....",
+"label.action.delete.system.service.offering": "Systemdienstangebot löschen",
+"label.action.delete.template": "Vorlage löschen",
+"label.action.delete.template.processing": "Vorlage wird gelöscht....",
+"label.action.delete.user": "Benutzer löschen",
+"label.action.delete.user.processing": "Benutzer wird gelöscht....",
+"label.action.delete.volume": "Volumen löschen",
```

Comment:
Datenträger löschen

Reply:
Festplatte, Disk, Laufwerk, Medium

Reply:
@svenvogel Can you help me check it. Thanks